src/config/index.ts

```diff
@@ -145,7 +145,7 @@ export const UPDATE_STATEMENT_VALIDATION_ERRORS_PAGE = "update-statement-validat
 // REMOVE TEMPLATES
 export const REMOVE_SOLD_ALL_LAND_FILTER_PAGE = "remove-sold-all-land-filter";
 export const REMOVE_IS_ENTITY_REGISTERED_OWNER_PAGE = "remove-is-the-overseas-entity-registered-as-the-owner";
-export const REMOVE_UNALLOWED = "remove-you-cannot-apply-to-remove-this-overseas-entity";
+export const REMOVE_CANNOT_USE_PAGE = "remove-cannot-use";
```

Comment:
Happy with the constant name change but wondering why value of the page name has changed. I think we normally keep these (close to) the name of the prototype page.

Reply:
I was following the naming of existing pages here rather than the prototype ones. e.g.
export const CANNOT_USE_PAGE = "cannot-use";

Reply:
Fair enough. What do others think?

Reply:
If so, we should probably update the other cannot use page as well.

Reply:
ok, if it's similar in registration then should be ok as is

Reply:
Yes, happy with this too.